### ISSUE TITLE
Using folders for scripts and creating scripts in folders is not working properly.

### ISSUE THREAD

**Issue by qlikq** [ironmansoftware/powershell-universal#596]
**Describe the bug**
Problems with viewing folder and its contents in Scripts pane. Text gets not aligned properly. repository folder name is positioned in a way that it written on the script names.

**To Reproduce**
1. Go to Scripts
2. Create a script, everything works
3. Create a folder test
4. Click on test folder, then click create script
5. This ends with an error. But it thought it would create a script inside that new folder.

**Expected behavior**
Actually i don't know what is expected behavior. Inside the documentation there is nothing mentioned about the folders for scripts. I thought that we can use this button : 'create folder' so one can group his scripts, but it looks that this does not work as intended. I can't create scripts in my folders which i create using that button. Formatting for column with repository folder name is off, and the text goes on the column where the script name [are.](url)

**Screenshots**

![Screenshot 2021-10-18 160853](https://user-images.githubusercontent.com/21127680/137815070-4c6546df-ac9a-4499-a6d0-e9aca3682b6b.png)

**Version**

Version Number: 2.4.0
Dashboard Framework (if applicable): 
Operating System: official docker container

**Additional context**

**Logs** 

//edit
https://1drv.ms/t/s!AnxAvnQjOaXIilIkhTiNJYpFPM9-?e=L2qDIl

**Comment by adamdriscoll** [comment]
Weird. I tried just now and it worked for me so I'll be curious to see screen shots and log. 

![image](https://user-images.githubusercontent.com/1256531/137787548-471dfdda-f472-427c-a069-a9aadba07368.png)

**Comment by qlikq** [comment]
> Weird. I tried just now and it worked for me so I'll be curious to see screen shots and log.

Hello Adam, thank you for reply. I have updated the main post entry with the links to log+picture.
Thank you for pasting screenshot. That's exactly what i would expect form it, but i didn't get to work it like this. If i would click on 'test' folder, and then click new script, it was also show a toast with 'Error'. 
Also it does this weird thing with formatting text so it does not get aligned properly within its column.
Tomorrow i will make fresh install and will give it another try. Any idea if that would matter, that i have mounted a volume for /data ? 
I am running windows10 , wsl2-ubuntu. On my windows i have c:\psufiles mounted as /data within the container. As per instructions from the docker hub container repo website.
I will make tomorrow another test, without mounting that directory as well, just to compare.

**Comment by qlikq** [comment]
hi @adamdriscoll , i have done another test, without mounting even my local resources folder into the container.
![image](https://user-images.githubusercontent.com/21127680/137921369-971e57df-0d39-4861-a92a-117b2ce11967.png)

I wanted to mention that, when i create Folders using the PSU website, i get error toast, but when i refresh the website  afterwards, it will show me the folders i created.
When i position my cursor on the 'Subfolder' directory, and click new script, it shows error, and script is not created. When i browse this folder : 'Subfolder' it is empty. When i browse folder called 'data' , i can find it, but this file name is : 
'Subfolder\subtest.ps1'  , including the quotes
When i create the first script file without creating folders, it is done correctly and the result is a file inside the directory:
root@2c6d05d8c000:/data/Repository# ls
basescriptinRepository.ps1  data  mythings  subfolder  Subfolder  test123

Not sure if it is my fault when i created the dockerfile, or some mistake. If anything my dockerfile is
FROM ironmansoftware/universal:latest
LABEL description="Universal - The ultimate platform for building web-based IT Tools"
EXPOSE 5000
VOLUME ["/data"]
ENV Data__RepositoryPath ./data/Repository
ENV Data__ConnectionString ./data/database.db
ENV UniversalDashboard__AssetsFolder ./data/UniversalDashboard
ENV Logging__Path ./data/logs/log.txt
ENV TZ Europe/Amsterdam
RUN apt-get install -y tzdata
ENTRYPOINT ["./home/Universal/Universal.Server"]

so that i can mount volume into /data , to make the container persistent.

Not sure if it is supposed to be like this that, there is another 'data' inside the /data/Repository

**Comment by qlikq** [comment]
I have tested now with the 'standard' image without even changing the dockerfile etc. and i was able to create new folder, but it was not possible for me to create a file inside of it, i was still getting the red 'Error' on the website.
This is what i see in the folder structure:
![image](https://user-images.githubusercontent.com/21127680/137924420-cfb416c5-75ab-4fbe-bbe3-a7965fd74c7f.png)

repository.ps1 was script created in the default folder. It worked just fine from the PSU website. Other wiles with the quote marks, were attempted to be crated using website PSU, but they in the website all i could see was red error. and i could not see new files. 
![image](https://user-images.githubusercontent.com/21127680/137924762-3c6c1d76-3d11-4cde-8651-0a056edbaf51.png)

**Comment by mabster** [comment]
Hey gang.

We've struck an issue of our own that seems to be related to this.

We create a Script Folder and create a Script inside it. The script executes just fine and we can see the pipeline output in the job.

We then create a Page and add a Table to it, pointing the table at the script we just created. The table spins for a few seconds and then says "no data".

The table works when pointing to a script in the root folder with the exact same PowerShell code.

So it seems that script folders might not be 100% there in terms of compatibility with other parts of the system.

This is a brand new install on my local machine. Version 2.4.0.

**Comment by qlikq** [comment]
I just tested the latest image, i think it was updated(i am not 100% sure). As of now, i can create new folders, and i am not getting a red error. There is still an issue when i want to create a script in that folder.
Possible idea: Maybe in case somebody creates a long folder name it should be visible as : 
verlongfolderna...   instead of creating that issue with text going into next column ?
![image](https://user-images.githubusercontent.com/21127680/139560827-bf490ee0-7c44-4313-825f-df8f4b627f8a.png)
